docs/release/release_notes.md

```diff
@@ -0,0 +1,21 @@
+## Release 3.3
+**In this release we update profiles to use the declaration eligibility field instead of the contribution.
```

Comment:
@mattwire I'm not sure I understand this change? Does this mean the field "eligible for gift aid" on the contribution is no longer used? I think users need this to find the contributions that should be eligible for gift aid to add to batch...

Reply:
The user needs to create a gift aid declaration - and there are situations where you want to create a gift aid declaration but not apply it to the contribution.  By displaying the declaration Yes/Yes past 4 years/No to the user they are explicitly updating the declaration and contributions will by default use that.  The contribution now has "Yes/No" because the 3rd option makes no sense at a contribution level.